fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/BestMvSelector.java

```diff
@@ -124,6 +142,13 @@ public int compare(CandidateContext context1, CandidateContext context2) {
             if (ret != 0) {
                 return ret;
             }
+
+            // larger is better
```

Comment:
why compare row number first then sort key ?

Reply:
changed to check sort key first